deploy/submariner/lighthouse_operator_cluster_role.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: submariner-operator:lighthouse
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - get
+      - list
+      - watch
+      - update
+  - apiGroups:
+      - lighthouse.submariner.io
+    resources:
+      - '*'
+    verbs:
+      - create
+      - get
+      - list
+      - watch
+      - update
+      - delete
```

Comment:
```suggestion
```

This part is already covered by deploy/lighthouse/cluster_role.yaml

and it's specific to "lighthouse", not the operator.

Reply:
Oh I was using submariner-operator service account and hence that was not available. Changed the service account.